libs/content/workspace/src/workspace.rs

```diff
@@ -810,6 +833,16 @@ impl Workspace {
             };
         }
     }
+    fn graph_called(&mut self) {
+        if !self.tabs.iter().any(|t| t.name == "graph") {
+            let id = Uuid::new_v4();
+            self.upsert_tab(id, "graph", "", false, true);
+            let mut graph = lockbookdata(&self.core);
```

Comment:
Ideally this looks a little more like
```suggestion
            let mut graph = KnowledgeGraph::new(&self.core);
```